### DISCUSSION TITLE
This website isn't social media

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54488] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

we don't need an algorithm recommending us repositories

**Comment by f-person** [comment]
While I do partially agree with the statement in regards to the recommendation algorithm (I think it's OK to have it, just not turned on by default), GitHub is a pretty social platform for me, I regularly read my feed through RSS to see what other folks have been doing here.
